src/SourceCode.Chasm/IO/ChasmConcurrencyException.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace SourceCode.Chasm.IO
+{
+    [Serializable]
+    public sealed class ChasmConcurrencyException : System.Data.DataException
+    {
+        #region Constructors
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ChasmConcurrencyException"/> class.
+        /// This is the default constructor.
+        /// </summary>
+        public ChasmConcurrencyException()
+        { }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ChasmConcurrencyException"/> class with the specified string.
+        /// </summary>
+        /// <param name="s">The string to display when the exception is thrown.</param>
+        public ChasmConcurrencyException(string s)
```

Comment:
Use parameter names from base type (message).

Reply:
I did - here's the base declaration: `public DataException(string s, Exception innerException);`